source/grid/tria_accessor.cc

```diff
@@ -1082,6 +1082,102 @@ set (const internal::Triangulation::TriaObject<structdim> &object) const
 
 
 
+template <>
+TriaAccessor<0,1,1>
+TriaAccessor<1,1,1>::vertex_accessor (const unsigned int i) const
```

Comment:
You should be able to do
```
template <spacedim>
TriaAccessor<0,1,spacedim>
TriaAccessor<1,1,spacedim>::vertex_accessor (const unsigned int i) const
{
    ...
}

template <int dim,int spacedim>
TriaAccessor<0,dim,spacedim>
TriaAccessor<1,dim,spacedim>::vertex_accessor (const unsigned int i) const
{
    ...
}

template <int structdim, int dim,int spacedim>
TriaAccessor<0,dim,spacedim>
TriaAccessor<structdim,dim,spacedim>::vertex_accessor (const unsigned int i) const
{
    ...
}
```
This should subsume all of the 10 current specializations.